cmd/ketch/configuration/configuration.go

```diff
@@ -87,3 +104,43 @@ func (cfg *Configuration) DynamicClient() dynamic.Interface {
 	}
 	return i
 }
+
+// GetKetchConfigObject returns the unmarshalled contents of the config.toml file
+func (cfg *Configuration) GetKetchConfigObject() KetchConfig {
+	return cfg.ketchConfig
+}
+
+// DefaultConfigPath returns the path to the config.toml file
+func DefaultConfigPath() (string, error) {
+	home, err := ketchHome()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(home, "config.toml"), nil
+}
+
+func ketchHome() (string, error) {
```

Comment:
Right now the exported functions of `Read` and `DefaultConfigPath` are only used in main, but in the future DefaultConfigPath might be needed if we need to write to the config.toml.

I'm sticking with how pack has it defined for now, since this functionality is largely borrowed from pack. However, something like adding the config path to the `Configuration` structure would alleviate the need for it to be public.